eman2/convert/dataimport.py

```diff
@@ -97,6 +99,39 @@ def importCoordinates(self, fileName, addCoordinate):
             else:
                 raise Exception('Unknown extension "%s" to import Eman coordinates' % ext)
 
+    def importCoordinates3D(self, fileName, addCoordinate):
+        from pwem import Domain
```

Comment:
this can go

Reply:
Yep, fixed in https://github.com/scipion-em/scipion-em-eman2/commit/3b805869019181e3e364f9837a73ef99f09c12dd